lib/kontena/machine/digital_ocean/cloudinit.yml

```diff
@@ -48,6 +52,12 @@ coreos:
         Name=weave*
 
         [Network]
+    - name: zz-default.network
+      runtime: false
+      content: |
+        # default should only match real network interfaces
+        [Match]
```

Comment:
I'm pretty sure that it does not work without overriding `zz-default.network`. Reason: this file matches every network interface.